x/subscription/keeper/keeper.go

```diff
@@ -64,3 +64,10 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) IsEpochStart(ctx sdk.Context) bool {
```

Comment:
drop this; have caller use epochstorage's IsEpochStart()

Reply:
done